code-challenges.js

```diff
@@ -52,3 +113,18 @@ var numberExampleRefactor2 = "hello"
 
 var numberExampleRefactor3 = 27
 var numberExampleRefactor4 = 24
+
+// copy/paste code from question #4
+// modify if statement to also include evaluator to check if "typeof" num1 and num2 are equal to a number data type
+// log with provided variables
+
+const subtractor = (num1, num2) => {
+    if(num1 > num2 && typeof num1 === "number" && typeof num2 === "number") {
+        return num1 - num2
+    } else {
+        return num2 - num1
+    }
+}
```

Comment:
This is good but if we pass in the wrong data type the else will still catch it and try to do mathematical operations. If we want the prompts expected output "Your input is not a number" we'll need to add that as an option for the condtional.